sdk/aws/ec2.py

```diff
@@ -87,47 +98,158 @@ def list_instances(self, params_dict=None):
         # return a dictionary that contains the instances_info array and the count of server instances
         return {"count": len(instances_info), "instances": instances_info}
 
-    def create_instance(
-        self, image_id, instance_type, key_name, security_group_id, subnet_id
-    ):
+    def _get_custom_vpc_id(self, vpc_name="openshift-sustaining-vpc"):
+        """
+        Get the custom VPC ID based on the VPC Name or Tag.
+        """
+        ec2_client = self.session.client("ec2")
+
+        try:
+            # Describe VPCs and filter by Name tag (or any other criteria)
+            response = ec2_client.describe_vpcs(
+                Filters=[{"Name": "tag:Name", "Values": [vpc_name]}]
+            )
+            vpcs = response.get("Vpcs", [])
+            if vpcs:
+                return vpcs[0]["VpcId"]
+            else:
+                raise Exception(f"No VPC found with name/tag '{vpc_name}'.")
```

Comment:
```suggestion
            vpcs = response.get("Vpcs", [])
            if not vpcs:
                raise Exception(f"No VPC found with name/tag '{vpc_name}'.")
            
            return vpcs[0]["VpcId"]
```
small improvement that will make easier to read the code

I have a question, do we really want to raise a exception if the VPC is not found?
I think it's important to us to discuss it because it will be important for SDK and the bot to handle the responses.
wdyt?